### PR TITLE
Added images-to-pdfs conversion script.

### PR DESCRIPTION
**Pull request by athityakumar**

### CODE REVIEW

File: scripts/images-to-pdfs/images-to-pdfs.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+require 'rmagick'
+
+# Get images directory as argument from terminal
+images_directory = ARGV[0]
+
+# Get all files from that directory
+all_images = Dir.entries(images_directory)
+
+#Segregate only the required the question paper images , just in case the directory contains other files
+all_images.keep_if { |a| ((a.start_with? "mid-") || (a.start_with? "end-")) && ((a.end_with? ".jpg") || (a.end_with? ".png") || (a.end_with? ".tif") || (a.end_with? ".gif") || (a.end_with? ".svg") || (a.end_with? ".bmp"))}
```
Comment (raivivek) [inline scripts/images-to-pdfs/images-to-pdfs.rb]: The line runs too long. Break it on multiple lines please?

File: scripts/images-to-pdfs/images-to-pdfs.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#Require this gawdly gem
```
Comment (raivivek) [inline scripts/images-to-pdfs/images-to-pdfs.rb]: This can go, perhaps! ;)

**Comment by raivivek** [comment]
Hey @athityakumar, Thanks for the code. I appreciate it. Could you address a few things before we get it merged? Perhaps, reduce the number of comments (but leave them in important parts), wrap the code inside a function so that we can import it directly inside the email parse script and use programmatically (than having to run as a script)? That is, option to call functions with arbitrary arguments and not hardcode command line input?

Dankon!

**Comment by athityakumar** [comment]
Updated the script! Can you have another look?

PS - We would need to create question-papers/images and question-papers/pdfs directories in this repository for the scripts to work together automatically.

ie, mfqp-source > (scripts > (images2pdfs) , (parser) , (upload)) , (images) , (includes) , (question-papers > (images) , (pdfs))

**Comment by raivivek** [comment]
Thanks!

**Pull request merged**